package-scripts.js

```diff
@@ -23,7 +23,7 @@ function test(testName, mochaParams) {
 module.exports = {
   scripts: {
     build: {
-      script: `browserify -e browser-entry.js --plugin ./scripts/dedefine --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' --ignore chokidar -o mocha.js`,
+      script: `browserify -e browser-entry.js --plugin ./scripts/dedefine --ignore "./lib/esm-utils.js" --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' --ignore chokidar -o mocha.js`,
```

Comment:
I propose to remove all `--ignore` options and add them in our "karma.config.js" file. `chokidar` is missing in latter, but in general these options are redundant.
And IMO some files as "bin/cli/*"  are missing completely.

Reply:
We also have a `browser` field in "package.json", plus the script above and "karma.config.js". I lost orientation, so lets wait with this.